Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/PodcastDataManager.swift

```diff
@@ -316,6 +316,9 @@ class PodcastDataManager {
     // MARK: - Updates
 
     func save(podcast: Podcast, dbQueue: FMDatabaseQueue) {
+        // Get the existing podcast to compare if folder is being changed
+        let existentPodcast = DataManager.sharedManager.findPodcast(uuid: podcast.uuid)
```

Comment:
My initial plan was to release these logs in `7.43`, however, due to this change I think some time out there in beta would be good, a time that unfortunately we don't have right now, so I'm targeting 7.44.